yt/data_objects/tests/test_firefly.py

```diff
@@ -0,0 +1,20 @@
+import nose
+
+from yt.config import ytcfg
+from yt.testing import fake_particle_ds, requires_module
+
+import tempfile
+
+@requires_module('firefly_api')
```

Comment:
You're going to need to add `firefly_api` to `tests/test_requirements.txt` so that it gets installed on the test machine.

Reply:
you mean avoiding the test completely doesn't count as passing it?? 

I've added firefly_api/0.0.2 to pypi (with my most recent changes) and put the module in test_requirements-- should be better now.

Reply:
I dunno, I always thought there was some merit to passing tests that way ...

![tape_fell_off](https://user-images.githubusercontent.com/89019/53972272-13b66900-40c4-11e9-8415-0f081abcd32f.jpg)